src/odemis/driver/piezomotor.py

```diff
@@ -326,17 +367,24 @@ def _doMoveAbs(self, f, pos):
                 usteps = int((steps_float - steps) * USTEPS_PER_STEP)
                 self.runMotorJog(self._axis_map[axname], steps, usteps, self._speed_steps[axname])
                 targets[axname] = None
-        self._waitEndMotion(f, targets)
-        # Leave target mode in case of closed-loop move
-        for ax in pos:
-            self.stopAxis(self._axis_map[ax])
-        self._updatePosition()
+        try:
+            self._waitEndMotion(f, targets)
+        except:
```

Comment:
Please, prefer `except Exception`. 
However, the whole block is not needed if you just do a raise. Maybe also add a logging.warning?

Reply:
I want the finally to be executed no matter what. So what it should be doing in case of an exception is just raise it, and then afterwards call the stop function on all axes to go back to a normal state. Is there a better way of doing it?

Reply:
If all you care about is the finally, it's possible to just write:
```python
try:
    blabla
finally:
    do it
```
(You don't need an except)